graph-manager-docs/source/managed-federation/setup.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Setting up managed federation
+sidebar_title: Setup
+---
+
+import ProjectConfigPanel from 'gatsby-theme-apollo-docs/shared/project-config-panel.mdx';
+
+This article describes how to set up Apollo Graph Manager for a data graph that uses [Apollo Federation](https://www.apollographql.com/docs/apollo-server/federation/introduction/).
+
+> As with all changes, you should first set up managed federation in a non-production environment, such as staging. To support this, you can use [variants](../schema-registry/#managing-environments-with-variants), which are distinct versions of the same data graph for different environments.
+
+## 1. Register your implementing services
+
+To enable managed federation, each of your data graph's [implementing services](https://www.apollographql.com/docs/apollo-server/federation/implementing-services/) must register its schema with Graph Manager:
+
+```mermaid
+graph BT;
+  subgraph " "
+  gateway([Gateway]);
+  serviceA[Products service];
+  serviceB[Reviews service];
+  serviceC[Inventory service];
+  end
+  graphManager{{Apollo Graph Manager}};
+  gateway --- serviceA & serviceB & serviceC;
+  serviceA & serviceB & serviceC -. apollo service:push .-> graphManager;
+  class graphManager secondary;
+```
+
+To get set up with schema registration, **do the following for each of your implementing services**:
+
+1. Complete the prerequisites in the dropdown below. **However, only create a single graph in Graph Manager**. All of your implementing services register their schema with the same graph.
+
+  <ProjectConfigPanel />
+
+2. In each service's `apollo.config.js` file, add a `name` field to the `service` object. This value must match the name of the graph in Graph Manager that your services belong to.
+
+    ```json{3}:title=apollo.config.js
+    module.exports = {
+      service: {
+        name: "docs-example-graph",
+        localSchemaFile: "schema.gql"
+      }
+    };
+    ```
+
+3. Obtain the following values, which are required for the `apollo service:push` command:
+
+    * The URL that your gateway will use to communicate with each service (e.g., `http://products-graphql.svc.cluster.local:4001/`)
+    * The name that will uniquely identify each service within your graph (e.g., `Products`)
+
+3. Now you can run the `apollo service:push` command with the necessary options from each service's root directory.
+
+    Assuming the example values in the previous step, that command looks like this:
+
+    ```
+    apollo service:push --serviceURL=http://products-graphql.svc.cluster.local:4001/ --serviceName=Products
```

Comment:
We should explain this will send a `query Introspekt { _ service { sdl } }` to localhost:4000/graphql by default (or provide a `--localSchemaFile` in the example)

Reply:
This is covered in step 1, which is an imported reusable component about setting up base configuration (including specifying either a local schema file or a URL (in which case introspection occurs))